data/en/cfadmin.json

```diff
@@ -1,26 +1,619 @@
-	"params": [
-		{"name":"action","description":"The administrative action to perform.","required":true,"default":"","type":"string","values":[]},
-		{"name":"type","description":"The admin context: `web` or `server`","required":true,"default":"","type":"string","values":["web","server"]},
-		{"name":"password","description":"The admin password.","required":true,"default":"","type":"string","values":[]}
-	],
-	"engines": {
-		"lucee": {"minimum_version":"", "notes":"", "docs":"http://docs.lucee.org/"}
-	},
-	"links": [
-		{
-			"title":"Old Railo CFAdmin Reference",
-			"url":"http://web.archive.org/web/20160430035417/http://www.getrailo.org/index.cfm/documentation/cfadmin-doc/",
-			"description":"Railo documentation for cfadmin tag."
-		}
-	],
-	"examples": [
-
-	]
-
+  "name": "cfadmin",
+  "type": "tag",
+  "syntax": "<cfadmin>",
+  "related": [],
+  "description": "Used internally by Lucee (and Railo) in the administrator to apply settings. This tag is considered undocumented / subject to change.",
+  "params": [
+    {
+      "name": "action",
+      "description": "The administrative action to perform.",
+      "required": true,
+      "default": "",
+      "type": "string",
+      "values": []
+    },
+    {
+      "name": "type",
+      "description": "The admin context: `web` or `server`",
+      "required": true,
+      "default": "",
+      "type": "string",
+      "values": [
+        "web",
+        "server"
+      ]
+    },
+    {
+      "name": "password",
+      "description": "The admin password.",
+      "required": true,
+      "default": "",
+      "type": "string",
+      "values": []
+    }
+  ],
+  "engines": {
+    "lucee": {
+      "minimum_version": "",
+      "notes": "",
+      "docs": "http://docs.lucee.org/guides/Various/cfadmin.html"
+    }
+  },
+  "links": [
+    {
+      "title": "Lucee Documentation",
+      "url": "http://docs.lucee.org/guides/Various/cfadmin.html",
+      "description": "Lucee documentation for cfadmin tag."
+    },
+    {
+      "title": "Old Railo CFAdmin Reference",
+      "url": "http://web.archive.org/web/20160430035417/http://www.getrailo.org/index.cfm/documentation/cfadmin-doc/",
+      "description": "Railo documentation for cfadmin tag."
+    }
+  ],
+  "examples": [
+    {
+      "title": "Create an admin collection",
+      "description": "",
+      "code": "<cfadmin\n action=\"collection\"\n type=\"web|server\"\n password=\"password\"\n collectionAction=\"create\"\n collection=\"#form.collName#\"\n path=\"#form.collPath#\"\n language=\"#form.collLanguage#\"\n remoteClients=\"arrayOfClients\">"
+    },
+    {
+      "title": "Delete an admin collection",
+      "description": "",
+      "code": "<cfadmin\n action=\"collection\"\n type=\"web|server\"\n password=\"password\"\n collectionAction=\"delete\"\n collection=\"#form.name[key]#\"\n remoteClients=\"arrayOfClients\">"
+    },
+    {
+      "title": "",
```

Comment:
Our tests fail if a title is empty. You can move the description to the title instead.